src/components/CollapsablePanel.sc.js

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+
+let ColapsableContainer = styled.div`
```

Comment:
Should we use also here double 'l' in `collapsable`? 
Also, it seems that in english it's `collapsible`? See: https://www.merriam-webster.com/dictionary/collapsable